ontobio/assoc_schema.py

```diff
@@ -0,0 +1,48 @@
+from marshmallow import Schema, fields, pprint
+
```

Comment:
So then do you use these to produce classes? Are these like "meta" classes? Or do you instantiate these directly?